lgc/patch/FragColorExport.cpp

```diff
@@ -49,6 +94,14 @@ namespace lgc {
 FragColorExport::FragColorExport(LLVMContext *context) : m_context(context) {
 }
 
+// =====================================================================================================================
+// Create a FragColorExport
+//
+// @param context : The context in which the FragColorExport will run.
+FragColorExport *FragColorExport::create(LLVMContext *context) {
+  return new FragColorExport(context);
+}
```

Comment:
This method doesn't seem useful, can we remove it?

Reply:
I semi followed what was done for the vertex fecth, which has a similar function.  However, because I did not separate this class in to an "implementation" class that is hidden, there is no actual need for this function.  I can delete it if you would like.

@trenouf What was the reason for the `VertexFetchImpl` class?  Since I don't have a FragColorExportImpl class, should I delete this function?

Reply:
My separation was just to make it a bit tidier by hiding the implementation.

Reply:
I've removed it.